source/extensions/filters/network/mongo_proxy/codec_impl.cc

```diff
@@ -395,6 +524,43 @@ void EncoderImpl::encodeReply(const ReplyMessage& message) {
   }
 }
 
+void EncoderImpl::encodeCommand(const CommandMessage& message) {
+  int32_t total_size = 16;
```

Comment:
You're cribbing existing style but would you mind instead defining a kMessageHeaderSize or something as part of the class so it's less of a magic number?

Reply:
Changed all magic numbers to constexpr. Compiler will most likely used constexprs in compile time to avoid integer calculations in run-time.